draft-ietf-dnsop-svcb-https.md

```diff
@@ -978,7 +982,7 @@ approach such as Happy Eyeballs {{!HappyEyeballsV2}}.
 When only "ipv4hint" is present, IPv6-only clients may synthesize
 IPv6 addresses as specified in {{!RFC7050}} or ignore the "ipv4hint" key and
 wait for AAAA resolution ({{client-behavior}}).  Recursive resolvers MUST NOT
```

Comment:
Another potential phrasing:

```suggestion
wait for AAAA resolution ({{client-behavior}}).  Note that any DNS64 AAAA record synthesis ({{Section 5.1.7 of ?RFC6147}}) applies only to AAAA queries, and trying to extrapolate it to synthesize ipv6hint parameters would likely result in DNSSEC and other compatibility issues.
```

Reply:
After thinking about this some more, I realized that my claim that this "applies to only AAAA queries" is not really correct, because we also want AAAA synthesis to apply to the Additional records returned from a SVCB query.  That would make this a positive, and therefore unambiguously normative, reference.

I've moved this discussion into a new subsection under Resolver Behavior.  Please review!

Reply:
I like it, thanks for the rewrite.